index.js

```diff
@@ -4,6 +4,9 @@
 
 const path = require('path');
 const fs = require('fs');
+const { spawn } = require('child_process');
+// eslint-disable-next-line node/no-unpublished-require
```

Comment:
:thinking:

Reply:
`chalk` should be in `dependencies` not `devDependencies`, then you can remove this line